devito/ir/equations/equation.py

```diff
@@ -27,10 +32,18 @@ def is_Scalar(self):
     def is_Tensor(self):
         return self.lhs.is_Indexed
 
+    @property
+    def is_Break(self):
+        return self._is_Break
+
     @property
     def is_Increment(self):
         return self._is_Increment
 
+    @property
+    def is_Reduce(self):
```

Comment:
what's, in practice, the difference between a Reduce and an Increment ?

Reply:
The difference is that this can be used to, for example, calculate other useful norms, for example, the Max norm. The Increment only works in additive norms such as the p-norm. I will leave the Reduce class for now and use only the p-norm in my notebooks, even though the Max norm is recommended for the analysis of the Poisson problem.

Reply:
The problem is, as pointed out by Nelson, that this class does not work with MPI. Before discussing that further, I will remove that from this pull request.